### DISCUSSION TITLE
Tile windows using long throw - is this possible?

### DISCUSSION THREAD

**Discussion by tweaknmod** [rxhanson/RectanglePro-Community#64] [Q&A]
I don't mean snapping the windows to the left and right, as is default. I have that set to regular throw.

What I'm wondering is if it's possible to use the long throw option to trigger MacOS' Tile mode, where the menubar and other UI are hidden, making the left and right apps occupy maximum screen space. 

Any guidance would be greatly appreciated.

**Comment by rxhanson** [comment]
At the moment, this is not possible. 

The only ways to trigger the tile mode would be to either programmatically execute the menu item, or to programmatically execute a keyboard shortcut that is associated with the menu item. Adding menu item / keyboard shortcut execution into Rectangle Pro is a little outside of my desired feature set at this time.

**Reply by rxhanson** [reply]
Also worth mentioning that there's no way to guarantee that the menu item for tiling and the associated keyboard shortcut are going to be identical for all users (given localization, user customization, etc). This means that I would be forced to implement the generic case of letting the user configure which menu item title or keyboard shortcut they want the window throw to execute in order to solve this issue, and the user would have to understand this when they are looking for this desired outcome.

**Comment by tweaknmod** [comment]
Totally fair. Thanks for taking the time to respond - love the app!
